select_builder.go

```diff
@@ -84,13 +91,45 @@ func (tx *Tx) SelectBySql(query string, value ...interface{}) SelectBuilder {
 	}
 }
 
+func (b *selectBuilder) changeTimezone(value reflect.Value) {
+	if b.timezone == nil {
+		return
+	}
+
+	v, t := extractOriginal(value)
```

Comment:
and what happens if you get a nil pointer or interface here?

Reply:
Nothing.